cmake/modules/RootBuildOptions.cmake

```diff
@@ -442,3 +442,12 @@ if(macos_native)
     message(STATUS "Option 'macos_native' is only for MacOS systems. Ignoring it.")
   endif()
 endif()
+
+# Print message saying with which versions of Python are used to build
+if(pyroot_experimental)
+  if(NOT Python3_Development_FOUND OR NOT Python2_Development_FOUND)
+    message(STATUS "Main Python used to build: ${PYTHON_VERSION_MAJOR}; PyROOT built for version ${PYTHON_VERSION_MAJOR}")
```

Comment:
This message and the one below need to be consistent with the format of the message for the one-python specified by the user:

```   message(STATUS "PyROOT will be built for version ${PYTHON_VERSION_MAJOR}") ```  and similarly for the case of two pythons below

Reply:
Here we said once that we want to notify which is the main python (used for TPython and other stuff) and which pythons we build pyroot with. So at least for the one below we need both the information.
Unless we want to avoid to specify which is the main python. Tbh I'm fine with everything.

Reply:
Ok so this one can be:
```
message(STATUS "PyROOT will be built for version ${PYTHON_VERSION_MAJOR}") 
```
since there is only one, no need to specify main (the same way we did not specify it in the message for the user-defined python).

And in the message for two Pythons, we can say:
```
message(STATUS "PyROOT will be built for versions ${PYTHON_VERSION_MAJOR} (Main) and ${OTHER_PYTHON_VERSION_MAJOR}"") 
```